### ISSUE TITLE
Instructions for introducing the event

### ISSUE THREAD

**Issue by karenword** [carpentries/instructor-training#1528]
In #1492 @ndporter notes that it would be nice to include instructions for turning on captioning for online events in our welcome episode. However, we do not currently include any logistical content related to introducing the event (e.g. break times, instructions for hand-raising etc.) in that episode, leaving it to Trainers to engage in introductory practices we describe later in the curriculum.

This kind of content might be good to add in an instructor note?

**Comment by brownsarahm** [comment]
Yes! that was also specifically the goal of  #1399 by way of the excellent feedback in #1395

**Comment by karenword** [comment]
Thanks @brownsarahm for the connections! I realized my paraphrasing was vague so I will specify that this issue focuses on instructions *to participants* regarding how to turn on captioning after it has been enabled by the host. #1492 adds an instructor note to ensure that captioning is enabled (which now should be default on all Carpentries rooms, but may be necessary in non-Carpentries rooms), and the Trainer can also remember to turn it on for themselves if they wish, but participants need instructions because they have to turn on captioning individually after it has been enabled.
